RNTrueLayerPaymentsSDK/js/NativeTrueLayerPaymentsSDK.ts

```diff
@@ -89,7 +89,7 @@ interface Spec extends TurboModule {
           separator?: string;
           uiElementBorder?: string;
         },
-        fontFamilyName: string
+        fontFamilyName?: string
```

Comment:
Would like mobile guys to take a look at this. This was required to get it to build, but absolutely could be wrong.